apps/emqx_bridge/src/emqx_bridge_api.erl

```diff
@@ -420,6 +420,9 @@ schema("/bridges/:id/:operation") ->
             ],
             responses => #{
                 204 => <<"Operation success">>,
+                400 => error_schema(
+                    'BAD_REQUEST', "Problem with configuration of external service"
```

Comment:
Maybe it is possible to be more specific? In what situation do we give back a BAD_REQUEST error? If you get this when the operation does not exist, then maybe something like this would be more informative?

"The operation does not exist."

Reply:
That was part of some other ticket and has been added already, see line below. We then return a `404` now.